jaxrs-spec/src/main/asciidoc/chapters/providers/_lifecycle_and_environment.adoc

```diff
@@ -35,13 +35,14 @@ process only those classes that are annotated with `@Provider`.
 
 [[services]]
 ==== Services
-On runtime initialization, an implementation MUST discover and register certain service providers in `jakarta.ws.rs.core.Configuration` runtime configurations.
-For each service X, where X is `jakarta.ws.rs.core.Feature` or `jakarta.ws.rs.container.DynamicFeature`, the set of providers to be registered MUST be determined by the method invocations below. However, when an `Application` subclass is present, and either `getClasses` or `getSingletons` returns a non-empty collection, those invocations for which the (possibly implicit) `ClassLoader` argument equals the `Application` subclass's `ClassLoader` MUST be skipped.
+On runtime initialization, an implementation MUST discover and register certain service providers in `jakarta.ws.rs.core.Configuration` runtime configurations, if and only if service loading is enabled.
+Service loading is enabled, except when there's an `Application` subclass whose `getProperties` method returns a map which contains the key `jakarta.ws.rs.loadServices` with a value of `Boolean.FALSE`.
```

Comment:
Historically, the spec document has been written without using contractions such as "there's". Could we use "there is" here?

Reply:
Sure, done.